vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.mobileapp.steps;
+
+import java.time.Duration;
+
+import org.jbehave.core.annotations.When;
+import org.vividus.bdd.steps.ui.validation.IBaseValidations;
+import org.vividus.mobileapp.action.TapActions;
+import org.vividus.ui.action.search.Locator;
+
+public class ActionSteps
+{
+    private final TapActions tapActions;
+    private final IBaseValidations baseValidations;
+
+    public ActionSteps(TapActions tapActions, IBaseValidations baseValidations)
+    {
+        this.tapActions = tapActions;
+        this.baseValidations = baseValidations;
+    }
+
+    /**
+     * Taps on <b>element</b> located by <b>locator</b> with specified <b>duration</b>
+     * @param locator locator to find an element
+     * @param duration between an element is pressed and released in
+     * <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO_8601</a> format
+     */
+    @When("I tap on element located by `$locator` with duration `$duration`")
```

Comment:
Won't it be confusing: having the tap step with duration only?

Reply:
The questions I got are:
- what duration will fit my app?
- Is it 0? 
- is it something else? 
- How to choose?
- If it 0 why I should even care?

Reply:
In this case I would prefer to gather some statistic which duration suits most apps by default, but currently there is not such statistic

Reply:
I could add composite with duration 1s but I would prefer to not add integration tests for such composites since the underlying component is already covered

Reply:
I propose to add one more step and use TouchAction.tap

Reply:
one more step is added